cloudformation/raid-substack.cfn.json

```diff
@@ -162,24 +162,54 @@
                 }
               ]
             },
-            "io1"
+            "gp3"
           ]
         },
         {
-          "Fn::Equals": [
+          "Fn::Or": [
```

Comment:
you forgot to update the Or in this stack :)

Reply:
👀 Oh, fixed